_includes/v21.2/sidebar-data/develop.json

```diff
@@ -274,22 +274,64 @@
           ]
         },
         {
-          "title": "Performance Best Practices",
+          "title": "Statement Tuning with <code>EXPLAIN</code>",
           "urls": [
-            "/${VERSION}/performance-best-practices-overview.html"
+            "/${VERSION}/sql-tuning-with-explain.html"
           ]
         },
         {
-          "title": "Use the EXPLAIN Statement",
+          "title": "Apply Statement Performance Rules",
           "urls": [
-            "/${VERSION}/sql-tuning-with-explain.html"
```

Comment:
@stbof, I think I misinterpreted this as a renaming. The old page is still around, just not listed here in the sidenav?

Reply:
Nevermind. I see it listed just above. Sorry for not looking more closely.